community/runc/APKBUILD

```diff
@@ -0,0 +1,60 @@
+# Maintainer: Bernhard J. M. Gruen <bernhard.gruen@googlemail.com>
+pkgname=runc
+pkgver="1.0.0_git20180309"
+pkgrel=0
+pkgdesc="CLI tool for managing OCI compliant containers"
+url="https://runc.io/"
+arch="all"
+license="Apache-2.0"
+makedepends="go linux-headers btrfs-progs-dev protobuf-dev"
+
+_commit="69663f0bd4b60df09991c08812a60108003fa340"
+_go_md2man_ver="1.0.8"
+
+source="runc-$pkgver.tar.gz::https://github.com/opencontainers/runc/archive/$_commit.tar.gz
+	go-md2man-$_go_md2man_ver.tar.gz::https://github.com/cpuguy83/go-md2man/archive/v$_go_md2man_ver.tar.gz
+	"
+	
+subpackages="
+	$pkgname-doc
+	"
+
+builddir="$srcdir"/runc-$_commit
+go_md2man_builddir="$srcdir"/go-md2man
+
+prepare() {
+	cd "$builddir"
+	mv -f vendor src
+	mkdir -p src/github.com/opencontainers
+	ln -s "$builddir" src/github.com/opencontainers/runc
+}
+
+build() {
+	# Required for building man-pages
+	ln -s  "$go_md2man_builddir"-"$_go_md2man_ver" "$go_md2man_builddir"
+	export GOPATH="$go_md2man_builddir"
+	export GOBIN="$GOPATH/bin"
+	export PATH="$GOBIN:$PATH"
+	cd "$go_md2man_builddir"
+	go get
```

Comment:
Will `go get` pull the exact same dependencies if I rebuild this a few months in the future or will it pick the latest version of the dependencies?